e2e/cypress/integration/02_global_filter.spec.js

```diff
@@ -54,25 +57,27 @@ describe('global projects filter', () => {
     cy.logout()
   })
 
-  it('shows allowed projects for non-admin', () => {
-    cy.login('/settings', nonAdminUsername)
-    // hide modal unrelated to test flow
-    cy.get('app-welcome-modal').invoke('hide')
+  // TODO can uncomment when we have a flag to remove legacy policies,
+  // which are currently allowing full project access to all users.
+  // it('shows allowed projects for non-admin', () => {
+  //   cy.login('/settings', nonAdminUsername)
+  //   // hide modal unrelated to test flow
+  //   cy.get('app-welcome-modal').invoke('hide')
 
-    cy.get('chef-sidebar')
-      .should('have.attr', 'minor-version')
-      .then((version) => {
-        if (version === 'v1') {
-          cy.get('[data-cy=projects-filter-button]').click()
+  //   cy.get('chef-sidebar')
+  //     .should('have.attr', 'minor-version')
+  //     .then((version) => {
+  //       if (version === 'v1') {
+  //         cy.get('[data-cy=projects-filter-button]').click()
 
-          const allowedProjects = [proj1, proj2];
-          cy.get('[data-cy=projects-filter-dropdown] chef-checkbox')
-            .should(($elements) => { expect($elements).to.have.length(allowedProjects.length) })
-          allowedProjects.forEach(project => {
-            cy.get('[data-cy=projects-filter-dropdown]').contains(project)
-          })
-        }
-      })
-    cy.logout()
-  })
+  //         const allowedProjects = [proj1, proj2];
+  //         cy.get('[data-cy=projects-filter-dropdown] chef-checkbox')
+  //           .should(($elements) => { expect($elements).to.have.length(allowedProjects.length) })
```

Comment:
Consider just commenting out L74-75 and leaving all the rest active. That way, it is at least testing something for a non-admin.

Reply:
I see some value in that but maybe not enough to be worth all the setup/teardown. Hopefully we'll get it up and running in full soon.